docs/planning/v3/charters/v3.3.0-external-tools.md

```
@@ -1,3 +1,19 @@
+> **Carried forward into this charter's scope on 2026-09-04**, from v3.2.0's close:
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)